hawk/src/main/java/com/orhanobut/hawk/HawkPreferenceChangedListener.java

```diff
@@ -0,0 +1,9 @@
+package com.orhanobut.hawk;
+
+/**
+ * Created by Florian Schweitzer on 2/15/2016.
+ */
+public interface HawkPreferenceChangedListener {
```

Comment:
we can remove Hawk and can use a more generic name. How about DataChangedListener?